### PR TITLE
magento/devdocs#: added example of using the modal widget

### PR DESCRIPTION
**Pull request by DashukVolodymyr**
## Purpose of this pull request

This pull request (PR)  adds an example of using the modal widget to the existing documentation.

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.4/javascript-dev-guide/widgets/widget_modal.html

### CODE REVIEW

File: src/guides/v2.3/javascript-dev-guide/widgets/widget_modal.md [hunk 1]
```diff
@@ -385,6 +385,45 @@ The following example shows how to initialize the modal widget and pass options
 </div>
 ```
 
+The following example shows a PHTML file using the script:
+
+```html
+<button id="button">Click me</button>
```
Comment (BarnyShergold) [inline src/guides/v2.3/javascript-dev-guide/widgets/widget_modal.md:391]: Would suggest encapsulating string literals in the PHTML with `$block->escapeHtml(__())` as this is best practice

**Review by BarnyShergold — CHANGES_REQUESTED** [review]

File: src/guides/v2.3/javascript-dev-guide/widgets/widget_modal.md (hunk 1 above)
Comment (DashukVolodymyr) [reply src/guides/v2.3/javascript-dev-guide/widgets/widget_modal.md]: Hi @BarnyShergold 
Thanks for your notice.
Your suggestion have been implemented!
Could you please verify this PR?
Thank you!

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
The other two literals - "Title" & "Content" have not been encapsulated.

**Comment by DashukVolodymyr** [comment]
> The other two literals - "Title" & "Content" have not been encapsulated.

Done!

**Review by BarnyShergold — APPROVED** [review]
Awesome

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
running tests

**Pull request merged**